bundle/manifests/cluster-logging.v5.2.0.clusterserviceversion.yaml

```diff
@@ -22,7 +22,7 @@ metadata:
     createdAt: 2018-08-01T08:00:00Z
     support: AOS Logging
     # The version value is substituted by the ART pipeline
-    olm.skipRange: ">=4.6.0-0 <5.0.0"
+    olm.skipRange: ">=4.6.0-0 <5.3.0"
```

Comment:
`5.2.0`

Reply:
it was `5.0.0` earlier. so `release-5.1` branch needs to update this as well. 
I will open a PR for 5.1